libnavigation-ui/src/test/java/com/mapbox/navigation/ui/camera/SimpleCameraTest.kt

```diff
@@ -0,0 +1,31 @@
+package com.mapbox.navigation.ui.camera
+
+import com.mapbox.navigation.ui.BaseTest
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class SimpleCameraTest : BaseTest() {
+
+    @Test
+    fun overview() {
```

Comment:
Could you add a regression test that calls `overview` twice with different route information to ensure that the value is recalculated?